src/views/backend/index.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/* @var $this yii\web\View */
+use yii\helpers\Html;
+?>
+<h1>Overview</h1>
+
+<p>
+  <?= Html::a('Templates', ['crud/contact-template']) ?>
+</p>
+
+<p>
+    <?= Html::a('Logs', ['crud/contact-log']) ?>
+</p>
+
+<p>
+    <?= Html::a('Export', ['export/index']) ?>
+</p>
```

Comment:
I would add an Yii:t instead of a plain string in Line [8,](https://github.com/dmstr/yii2-contact-module/pull/4/files#diff-0a4147502df697c3279afd759dced8bfde6d1b25818a5bc1bd6aa0702d013cb2R8) [12](https://github.com/dmstr/yii2-contact-module/pull/4/files#diff-0a4147502df697c3279afd759dced8bfde6d1b25818a5bc1bd6aa0702d013cb2R12) & [16](https://github.com/dmstr/yii2-contact-module/pull/4/files#diff-0a4147502df697c3279afd759dced8bfde6d1b25818a5bc1bd6aa0702d013cb2R16)